### DISCUSSION TITLE
How to align text of barcode

### DISCUSSION THREAD

**Discussion by D3LE0N** [micjahn/ZXing.Net#536] [Q&A] [answered]
How to I can align text to start of the generated barcode?

![image](https://github.com/micjahn/ZXing.Net/assets/36517969/6918a4fd-ce80-4d32-8b5a-a33ebc135e7a)

Like the next example:

![image](https://github.com/micjahn/ZXing.Net/assets/36517969/66c4e1fb-eb85-4943-a756-b7b442edebe8)

Thanks for the responses.

Greetings from Guatemala city!

**Comment by micjahn** [comment, marked as answer]
There is no built-in option which you can set.
You have to write your own renderer or you generate the barcode image without the text and add it manually.
